packages/query-engine/source/app.ts

```diff
@@ -4,16 +4,23 @@ import logger from "morgan";
 import cors from "cors";
 
 import { queryEngine } from "./rest";
+import { BadRequestError, NotFoundError, UnauthorizedError } from "./utils";
 
 const initialize = async () => {
     const app = express();
     app.use(cors());
     app.use(bodyParser.json());
 
+    // Attach router
     const router = express.Router();
     queryEngine.attachRoutes(router);
     app.use("/api/v1", router);
 
+    // Attach error handlers
+    app.use(BadRequestError);
+    app.use(NotFoundError);
+    app.use(UnauthorizedError);
```

Comment:
1. In a way, it makes sense to split the error handling. But then again, why not combine all the error handlers into a single error handler? I'm in favor of combing all the error handlers together.
2. Why are you using PascalCase for middleware? Further, people will get confused with exception classes from `@hypertool/common` with the same name.